src/cnaas_nms/api/jobs.py

```diff
@@ -30,7 +30,7 @@ def get(self):
         data = {'jobs': []}
         total_count = 0
         with sqla_session() as session:
-            query = session.query(Job, func.count(Job.id).over().label('total'))
+            query = session.query(Job, func.count(Job.id).over().label('total')).order_by(Job.id)
```

Comment:
The next row will call build_filter which also optionally adds .order_by() , how will SQLalchemy handle two order_by for the same query?
Another idea might be to set order_by id in the build_filter if no option was specified by the user (but since build_filter is a generic function, and all tables might not have an id column, we should also check that the column exists before trying to sort on it in there)

Reply:
Two order_by don't seem to be a problem, the jobs where returned in order and nothing seems to be broken. However, I fully agree that fixing it in build_filter is the "right" thing to do, but also more work.

Reply:
I sent another commit that does this via the generic function instead, and it seems to be working for /devices as well using this method

Reply:
I can't add you as reviewer but please merge if you think it looks good :P